src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncSpec.cs

```diff
@@ -260,7 +262,8 @@ public void A_Flow_with_SelectAsync_must_finish_after_task_failure()
                     .Grouped(10)
                     .RunWith(Sink.First<IEnumerable<int>>(), Materializer);
                 
-                t.AwaitResult().Should().BeEquivalentTo(new[] {1, 2});
+                var complete = t.ShouldCompleteWithin(3.Seconds());
```

Comment:
This Task isn't awaited?

Reply:
Yes, but should (1) be `var completed = await t.ShouldCompleteWithin(5.seconds));`? The `await` keyword is missing.

Reply:
`ShouldCompleteWithin()` returns a `Task` which needs to be `await`-ed. Please check your code.